README.md

```diff
@@ -108,6 +108,14 @@ should receive the Rows from in and may send any number of Rows to out. It
 should not return until it has finished all work (received all the Rows it's
 going to receive, sent all the Rows it's going to send).
 
+## Development
+Tests expects the files to be in `$GOPATH/src/gopkg.in/clever/optimus.v3` to work properly
```

Comment:
`clever` needs to be capitalized. Otherwise, it only works if you're on a case-insensitive file system.

Also, can we rephrase for clarity. Something like: "For the tests to work properly, you must be developing inside your GOPATH (i.e. `$GOPATH/src/gopkg.in/Clever/optimus.v3`)"